front/public/locales/fr/stdcm.json

```diff
@@ -26,7 +26,7 @@
     },
     "stopCalculation": "Arrêter le calcul"
   },
-  "spaceSpeedGraphic": "Graphique Espace-Vitesse",
+  "spaceSpeedGraphic": "Graphique Espace Vitesse",
```

Comment:
Maybe keep only 'speedSpaceChart' so we don't have to names for the same traduction

Reply:
done